app/controllers/application_controller.rb

```diff
@@ -2,4 +2,19 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def search_criterias
```

Comment:
@Murzic I've moved this method to lib from where this method could be called and used by other controllers after very small change in arguments